code/_onclick/hud/screen_objects.dm

```diff
@@ -23,6 +23,8 @@
 	master = null
 	return ..()
 
+/obj/screen/orbit(atom/A, radius, clockwise, rotation_speed, rotation_segments, pre_rotation, lockinorbit)

```

Comment:
if you're having the signatures, put the default args back, if you're not putting the defaults back, remove the signatures.

your call, pick one.